src/style.css

```diff
@@ -1 +1,97 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  width: 100%;
+}
+
+:root {
+  --turquoise-color: #00acdc;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.nav_logo {
+  width: 40px;
+  height: 40px;
+  box-sizing: border-box;
```

Comment:
Not resolved. You need remove box-sizing: border-box; from all places and add it to * selector.